builtin/providers/aws/resource_aws_ecs_service.go

```diff
@@ -292,16 +289,48 @@ func resourceAwsEcsServiceDelete(d *schema.ResourceData, meta interface{}) error
 		Timeout:    5 * time.Minute,
 		MinTimeout: 1 * time.Second,
 		Refresh: func() (interface{}, string, error) {
-			log.Printf("[DEBUG] Checking if ECS service %s is INACTIVE", d.Id())
+			// The DescribeServices returns "DRAINING" for several hours/days
+			// even when the service is actually deleted and no longer visible
+			// in the cluster
```

Comment:
:+1: I appreciate the comment here!

I can report this to AWS, unless you've done it already.